src/components/transaction/TransactionForm.js

```diff
@@ -1,260 +1,206 @@
-import React, { Fragment, useEffect, useState, useContext } from 'react';
-import TransactionContext from '../../context/transaction/transactionContext';
-import Papa from 'papaparse';
-
-const TransactionForm = () => {
-  const transactionContext = useContext(TransactionContext);
-
-  const { addTransaction } = transactionContext;
+import React, { Fragment, useEffect, useState } from "react";
+import uuid from "uuid";
+import moment from "moment";
+import Papa from "papaparse";
+
+const TransactionForm = (props) => {
+  const buildTransaction = (transaction) => {
```

Comment:
is there a distinction we can make between the function name and the parameter name? like `rawTransactionData` or something. both are named `transaction` currently

Reply:
Yes. This was messing with me when I was updating the references to transactions to be props.